src/common/search/hooks/usePlayFilter.js

```diff
@@ -0,0 +1,40 @@
+import { fetchFilterData } from "common/services/request/query/fetch-filter-data";
+import { submit_multi } from "../../services/request";
+import { useEffect, useState } from "react";
+import { getAllTags as extractTags } from "meta/play-meta-util";
+
```

Comment:
Provide a description of what is this hook supposed to do.

Reply:
ok